### DISCUSSION TITLE
求解Tree组件里面使用异步加载数据时 我如何不点击Tree上的展开图标从而进行请求数据

### DISCUSSION THREAD

**Discussion by AlexXiaoLiu** [ant-design/ant-design#30801] [Q&A]
技术栈是react+antd 需求是有一个Tree 上面节点是异步加载的 在当前树组件里面我有一个事件订阅我会通过其他组件来触发这个事件订阅，当我树组件接收到触发的时候，我会刷新当前树的某些节点来请求他的子级节点并定位，有类似的解决方案么 求求了 孩子太苦了 61当天还要苦苦思考这个需求的实现 呜呜呜

**Comment by yoyo837** [comment]
请查看文档，支持的,state管理数据，第三者触发加载就好了。

**Reply by AlexXiaoLiu** [reply]
请教一下 您是指当我需要重新触发的时候 只需要去触发loadData相对应的函数即可吗

**Reply by yoyo837** [reply]
建议系统学习一下React.

**Reply by AlexXiaoLiu** [reply]
![006APoFYly1gfeby8zfo9j30ad096jrj](https://user-images.githubusercontent.com/24892907/120265903-7d3d7600-c2d3-11eb-87a9-a64cad267c15.jpg)
我的我的  我是傻逼   谢谢大佬

**Reply by yoyo837** [reply]
不是啥大佬，主要是不了解你情况，也没法帮你写代码。或者你可以提供个在线demo来交流。
